pkg/cluster/context.go

```diff
@@ -189,6 +198,15 @@ func (c *Context) Create(cfg *config.Config, options ...CreateOption) error {
 		return err
 	}
 
+	if !cc.InstallKubernetes {
+		fmt.Printf(
+			"Cluster creation complete. To setup Kubernetes:\n\n"+
```

Comment:
so i don't think anybody knows what Kubernetes is.
i think it's a Framework the produces binaries that expose an API, not the distro or a kernel as per the popular argument.

having said that we already install Kubernetes by copying it's binaries to the nodes.
would it make sense to call the option "SetupKubernetes" instead of "InstallKubernetes"?

Reply:
yeah, kubernetes is already "installed" to the nodes by this point more or less, setup makes more sense